src/components/chat-item/chat-prompt-input.ts

```diff
@@ -312,10 +322,47 @@ export class ChatPromptInput {
 
           this.quickPickOpen = true;
         }
+      } else if (navigationalKeys.includes(e.key)) {
+        const direction = e.key === KeyMap.ARROW_UP ? 'up' : 'down';
+        this.clearTextArea();
+
+        if (this.userPromptHistoryIndex === -1) {
+          this.userPromptHistoryIndex = this.userPromptHistory.length;
+        }
+
+        if (direction === 'up') {
```

Comment:
Why not using the KeyMap.ARROW_UP or _DOWN directly instead? I don't see any reason to put it to one more variable again.